src/screen.py

```diff
@@ -24,12 +26,38 @@ def __init__(self, monitor: int = 0, wait: int = 20):
         self._config = Config()
         self._monitor_roi = self._sct.monitors[monitor_idx]
         # auto find offests
+        self.found_offsets = False        
+        if self._config.general["find_window_via_win32_api"] :
+            Logger.debug("Using WinAPI to search for window under D2R.exe process")    
+            position = self.find_window_via_winapi()
+            if position is None:
+                Logger.debug("Can't find any window owned by D2R.exe falling back to matching via assets. Make sure D2R is in focus and you are on the hero selection screen")
+                position = self.find_window_via_assets(wait)
+        if position is not None:
```

Comment:
well now this position will not be initialize here though unless I'm missing something, have you tried running this code with the config disabled? is it fine?

Reply:
You're right let me get that fixed

Reply:
Tested 4 scenarios, fixed a bug with position not being initialized

Reply:
thanks I find it clearer now